### ISSUE TITLE
BUG - [Sev-3] - [Android] - Information alert for submitting mileage only travel claim is not read in logical order with Talkback on Android

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#13379]
## What happened?

When screen reader lands on the information alert for submitting mileage only travel claim, it is not read in a logical order.
First, the entire alert is highlighted but the middle is read. Then skip back to the top to read the header on the next swipe, and finally skips to the button on the last swipe.
## Specs:

- Device: Android.
- OS: Android
- User Account: Demo is fine
- Accessibility State: Talkback is on.

## Steps to Reproduce

1. Navigate to Past Appointments and find an appointment with a blue travel claim flag/bubble.
2. Enable Talkback
3. Tap to go to details.
4. Swipe to the blue information alert and notice the entire alert is highlighted but only the middle text (body) is read.
5. Swipe again and see the header is read.
6. Swipe again and see the button is read.

## Desired behavior

Should be top-down.  First the header, then the body, then the button.

## Acceptance Criteria

Read top-down in logical order.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: Low 

- 3 - Low

## Screen shot(s) and additional information

<img width="1164" height="678" alt="Image" src="https://github.com/user-attachments/assets/d44225c8-f1bd-4576-9fb5-b326da2bd037" />

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [x] Desired behavior is added
- [x] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
